src/seedu/addressbook/data/Tagging.java

```diff
@@ -0,0 +1,21 @@
+package seedu.addressbook.data;
+
+import seedu.addressbook.data.person.Person;
+import seedu.addressbook.data.tag.Tag;
+
+public class Tagging {
+	Person person;
+	Tag tag;
+	boolean isAdd;
+	
+	public Tagging(Person person, Tag tag, boolean isAdd){
+		this.person = person;
+		this.tag = tag;
+		this.isAdd = isAdd;
```

Comment:
Indentation problem here?. Our coding standard requires you to use 4 spaces instead of tabs. You can configure Eclipse to convert tabs to spaces automatically.